calm_adapter/calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/CalmXmlResponse.scala

```diff
@@ -75,7 +75,10 @@ object CalmSearchResponse {
       .toEither
 }
 
-case class CalmSummaryResponse(val root: Elem, retrievedAt: Instant)
+case class CalmSummaryResponse(
+  val root: Elem,
+  retrievedAt: Instant,
+  suppressedFields: List[String] = Nil)
```

Comment:
nit: use `Set[String]` instead of `List[String]`. Better if you're doing `elem in collection` type lookups.

Reply:
Obvs `O(n)` vs `O(1)` not gonna be an issue in this case (if there is only a single item in the list as currently I would actually expect the list implementation to be more efficient as is just an equals and doesn't involve hashing). Maybe set is more descriptive of the data we want so happy to change for that reason

Reply:
Yeah for sure, I feel it's more a good habit than strictly necessary in this case. I don't feel strongly either way.

Reply:
:+1: